src/components/Sum.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Sum({ a, b }) {
+  return (
+    <h2>
+      {`Sum of ${a} and ${b} is ${a + b}`}
```

Comment:
```suggestion
      {`Sum of {a} and {b} is {a + b}`}
```
It works without **$**